engine/flink/management/src/it/resources/docker/Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM flink:1.16.0-scala_${scala.major.version}-java11
+FROM flink:1.16.0-scala_2.12-java11
 
 COPY entrypointWithIP.sh /
 COPY conf.yml /
+COPY log4j-console.properties /
 
 #TODO: figure out users...
 
 USER root
 RUN chown flink /entrypointWithIP.sh
 
 RUN chown flink /conf.yml
+RUN chown flink /log4j-console.properties
 RUN chmod +x /entrypointWithIP.sh
```

Comment:
`RUN rm lib/flink-scala_2.12-1.16.0.jar`

I wonder if adding this + downloading [scala 2.13 library](https://search.maven.org/remotecontent?filepath=org/scala-lang/scala-library/2.13.10/scala-library-2.13.10.jar) would be sufficient for scala 2.13 to work?

Reply:
it worked